src/style.css

```
@@ -1 +1,71 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.site-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
```

it'd be better to use shorthand
```suggestion
  padding: 0 50px;
```